protocols/ping/src/handler.rs

```diff
@@ -148,7 +148,7 @@ pub struct Handler {
     /// Tracks the state of our handler.
     state: State,
     /// The peer we are connected to.
-    peer: PeerId,
+    _peer: PeerId,
```

Comment:
We can remove this field if it is no longer needed. The `Handler` is crate-private so we should be able to do that without breaking functionality.